e2e/iam/atlas_users_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e iam
+
+package iam_test
+
+import (
+	"encoding/json"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestUsers(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	//t.Run("invite", func(t *testing.T) {
+	//	cmd := exec.Command(cliPath,
+	//		iamEntity,
+	//		usersEntity,
+	//		"invite",
+	//		"--username",
+	//		"test@gmail.com",
+	//		"--password",
+	//		"3ert34221993.",
+	//		"--orgRole",
+	//		"5e429e7706822c6eac4d5970:ORG_MEMBER",
+	//		"--email",
+	//		"test@gmail.com",
+	//		"--firstName",
+	//		"test",
+	//		"--lastName",
+	//		"test",
+	//		"--country",
+	//		"IE")
+	//	cmd.Env = os.Environ()
+	//	resp, err := cmd.CombinedOutput()
+	//
+	//	if err != nil {
+	//		t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+	//	}
+	//})
```

Comment:
I am not sure how to test this command since we do not have delete user for atlas